src/interfaces/IMetaMorpho.sol

```diff
@@ -95,8 +95,14 @@ interface IMetaMorphoBase {
     /// @notice Revokes the pending cap of the market defined by `id`.
     function revokePendingCap(Id id) external;
 
-    /// @notice Submits a forced market removal from the vault, potentially losing all funds supplied to the market.
+    /// @notice Submits a forced market removal from the vault, losing all funds supplied to the market.
+    /// @notice Funds can always be recovered as a donation by withdrawing from this market (using `reallocate`), but
+    /// funds will be distributed pro-rata to the shares at the time of withdrawal, not at the time of market removal.
+    /// @notice This forced removal is expected to be used as an emergency process in case a market constantly reverts.
+    /// To softly remove a sane market, the curator role is expected to bundle a reallocation that empties the market
+    /// first (using `reallocate`), followed by the removal of the market (using `updateWithdrawQueue`).
     /// @dev Warning: Submitting a forced removal will overwrite the timestamp at which the market will be removable.
+    /// @dev Warning: Removing a market with non-zero supply will instantly impact the vault's price per share.
```

Comment:
nb: donating to the vault too

Reply:
And bad debt realization on an enabled market too, but it's ok to not mention it here because they are not directly related to this `submitMarketRemoval` function